mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java

```diff
@@ -247,11 +247,20 @@ public void onChargeCreatedFailure(String errorMessage) {
         Toaster.show(rootView, errorMessage);
     }
 
+    private int recyclerCount() {
+        int count = 0;
+        if (mChargesNameListAdapter != null) {
+            count = mChargesNameListAdapter.getItemCount();
+        }
+        return count;
+    }
     @Override
     public void showEmptyCharges() {
-        ll_error.setVisibility(View.VISIBLE);
-        mNoChargesText.setText(getResources().getString(R.string.message_no_charges_available));
-        mNoChargesIcon.setImageResource(R.drawable.ic_assignment_turned_in_black_24dp);
+        if (recyclerCount() == 0) {
```

Comment:
could just be ```if (mChargesNameListAdapter.getItemCount() == 0) {```.

Reply:
I tried that, but it was not correct as when you go to the client with no charges the message doesn't appear.